about/policy/license-disclaimer/index.md

```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
 ### License Attribution
 {: .text-left}
 
-UGRC requests that where possible and practical, **appropriate attribution** shall be made by users of this website, data, or any other services provided by UGRC with the following criteria:
+UGRC requests that wherever possible and practical, you include **appropriate attribution** when using data or services provided by UGRC in your maps, web applications, or other content. This includes providing credit to UGRC and indicating you made any changes to the data.
+
+For referenced data hosted by other organizations (such as data from the Utah Geological Survey), please contact the owner for attribution guidelines.
+
+**Detailed references** in a paper or book should follow your particular style guide. Please include the last updated date if it is available in the metadata or the date you accessed/downloaded the data if not. For example, this is what it could look like in an MLA-esque format:
+
+- State Geographic Information Database. _Utah Address Points_. (Feature class). Salt Lake City, UT: Utah Geographic Resource Center, July 2021. Geospatial point data. 27 July 2021. https://gis.utah.gov/data/location/address-data/.
```

Comment:
**praise**: this is cool!

Reply:
https://twitter.com/natfriedman/status/1420122675813441540?s=20 should we add somethign like this to our website repo?

Reply:
Gotta have something to show for all those college papers :)